### DISCUSSION TITLE
Evaluating Mitosis for Cross-Platform UI Development with Angular and React (Native)

### DISCUSSION THREAD

**Discussion by jfbaro** [BuilderIO/jsx-lite#1473] [General]
Hello there!

I've been exploring Mitosis and shared it with a friend who raised a couple of concerns. He mentioned that Mitosis might not support some of the latest Angular 18 features and could potentially lead to another lock-in.

Currently, we have two teams and UI libraries: one for React Native (mobile) and one for Angular (web). With upcoming acquisitions, we might soon need to support React.js (web) as well, which would require creating (and maintaining) yet another UI library.

Given this situation, I have a few questions:

- Is Mitosis gaining traction as a popular cross-platform UI library (generator) in the JavaScript community?
- Can we rely on Mitosis to fully support Angular, ensuring that it remains one of the top frameworks in Mitosis' target list and gets continuous community support?

Thanks for your insights!

**Comment by samijaber** [comment]
> Is Mitosis gaining traction as a popular cross-platform UI library (generator) in the JavaScript community?

We don't have any hard numbers, but we are hearing of more and more folks/companies using Mitosis for their design systems.

> Can we rely on Mitosis to fully support Angular, ensuring that it remains one of the top frameworks in Mitosis' target list and gets continuous community support?

We use Mitosis to generate our Gen2 SDKs, and are currently actively improving the Angular generator as part of creating a new Gen 2 Angular SDK. So it is undergoing a lot of improvements.

At the same time, Mitosis has an enormous surface area (given the large number of web frameworks that require generators). We ultimately need to rely on the community to provide support and keep the generators up-to-date with the latest features as new versions come out.

If a generator is providing sub-optimal output, please raise a Github issue outlining the problem, and if possible, offer your help in contributing a fix!
